alex/applications/PublicTransportInfoCS/data/train_names.expanded.txt

```diff
@@ -0,0 +1,178 @@
+albert einstein	albert einstein
```

Comment:
It should be the same as for the stops - so yes in reality it should be probably all of the inflected forms. I will need to fix this along with expanding the list of train names in a next pull request. The train name feature is not complete yet (don't know where to reliably take the train names).